frontend/src/queries/useBakerListQuery.ts

```diff
@@ -67,6 +67,11 @@ const BakerQuery = gql<BakerListResponse>`
 								finalizationCommission
 								bakingCommission
 							}
+							paydayCommissionRates {
+								transactionCommission
+								finalizationCommission
```

Comment:
Finalization commission goes away. It is 0 and should not be shown.

Reply:
Removed (was not used)